microsoft-edge/devtools-guide-chromium/accessibility/forced-color-simulation.md

```diff
@@ -0,0 +1,100 @@
+       Open the **Rendering Tools**  
+    :::image-end:::  
+
+1.  In **Rendering**, choose the drop down in **Emulate CSS media feature forced-colors**.  
+    
+    :::image type="complex" source="../media/accessibility-forced-colors-rendering-dropdown.msft.png" alt-text="The forced colors menu on the Rendering drawer" lightbox="../media/accessibility-forced-colors-rendering-dropdown.msft.png":::
+       The forced colors menu on the **Rendering** drawer  
+    :::image-end:::  
+    
+1.  Choose either `No emulation` or `forced-colors: active`.  
+    
+1.  The current page in the main window now displays the selected simulation.  
+    
+    :::row:::
+       :::column span="":::
+          :::image type="complex" source="../media/accessibility-forced-colors-rendering-no-modification.msft.png" alt-text="Display a web site without high contrast simulation" lightbox="../media/accessibility-forced-colors-rendering-no-modification.msft.png":::
+             Display a web site without high contrast simulation  
+          :::image-end:::  
+       :::column-end:::
+       :::column span="":::
+          :::image type="complex" source="../media/accessibility-forced-colors-rendering-dropdown-active.msft.png" alt-text="Display a web site in high contrast simulation" lightbox="../media/accessibility-forced-colors-rendering-dropdown-active.msft.png":::
+             Display a web site in high contrast simulation  
+          :::image-end:::  
+       :::column-end:::
+    :::row-end:::
+    
+## Use the Command Menu  
+
+1.  [Open the Command Menu][DevtoolsGuideChromiumCommandMenuIndexOpen].  
+1.  Type `forced-c`, select either **Emulate CSS forced-colors: active** or **Do not emulate CSS forced-colors**  and then select `Enter`.  
+    :::row:::
+       :::column span="":::
+          :::image type="complex" source="../media/accessibility-forced-colors-command-menu-emulate.msft.png" alt-text="Turn on high contrast simulation using the command menu" lightbox="../media/accessibility-forced-colors-command-menu-emulate.msft.png":::
+              Turn on high contrast simulation using the command menu  
+          :::image-end:::  
+       :::column-end:::
+       :::column span="":::
+          :::image type="complex" source="../media/accessibility-forced-colors-command-menu-donotemulate.msft.png" alt-text="Turn off high contrast simulation using the command menu" lightbox="../media/accessibility-forced-colors-command-menu-donotemulate.msft.png":::
+             Turn off high contrast simulation using the command menu  
+          :::image-end:::  
+       :::column-end:::
+    :::row-end:::  
+
+View and change your CSS like any other web page.  For more information, see [Get Started With Viewing And Changing CSS][DevtoolsGuideChromiumCssIndex].  
+
+<!-- links --> 
+
+[DevtoolsGuideChromiumMain]: ../../devtools-guide-chromium.md "Microsoft Edge (Chromium) Developer Tools | Microsoft Docs"  
```

Comment:
```suggestion
[DevtoolsGuideChromiumMain]: ../index.md "Microsoft Edge (Chromium) Developer Tools | Microsoft Docs"  
```
According to my calculations, go up 1 dir, then open index file.  Check this link after commit & build.  To fix build Warning.